app/models/lightning.py

```diff
@@ -950,7 +950,10 @@ class LnInfo(BaseModel):
     commit_hash: str
 
     # The identity pubkey of the current node.
-    identity_pubkey: str
+    identity_pubkey: str = Query("the nodes pubkey")
```

Comment:
If you use a Query a default value is recommended. Either and empty string or just None if optional.
```python
Query("" ,"The nodes lightning pubkey. Currently an empty string on platforms other than RaspiBlitz.")
```

Reply:
comitted